egui/src/widgets/slider.rs

```diff
@@ -324,9 +355,20 @@ impl<'a> Slider<'a> {
             } else {
                 self.value_from_position(position, position_range.clone())
             };
+
             self.set_value(new_value);
         }
 
+        if response.hover_pos().is_some() {
+            let prev_value = self.get_value();
+            let input = ui.input();
+            let delta = (input.scroll_delta.x + input.scroll_delta.y) as f64;
+            if let Some(scroll_step) = self.scroll_step {
+                // Scroll speed decided by consensus: https://github.com/emilk/egui/issues/461
+                self.set_value(prev_value + (delta / 50_f64) * scroll_step);
```

Comment:
This isn't very robust considering how many backends egui has.

If we want to handle discreet scroll steps (like those from a mouse), then we should consider adding those as separate events in egui. For instance, scrolling on one step on a mouse could produce `scroll_delta.y ≈ 50.0` and `scroll_detla_steps.y == 1`. Scrolling with a touch-pad would produce a smooth `scroll_delta.y` but no `scroll_detla_steps`.

Reply:
I agree, i think it's worth just ignoring input from the touchpad or similar devices, but to make such a change, you will also have to make changes to the backends.